src/routes/icgc-import.ts

```diff
@@ -0,0 +1,17 @@
+import * as express from 'express';
+import multer from 'multer';
+import { wrapAsync } from '../middleware';
+import submissionAPI from '../submission/submission-api';
+
+const router = express.Router({ mergeParams: true });
+const upload = multer({ dest: '/tmp' });
+
+router.post(
+  '/preprocess/:programId',
+  upload.fields([{ name: 'donor' }, { name: 'specimen' }, { name: 'sample' }]),
+  wrapAsync(submissionAPI.processLegacyIcgcData),
```

Comment:
Just checking, so the idea is to first use `/submission/icgc-import/preprocess/{programId}preprocess/:programId` then take the output and use `/submission/icgc-import/`??

Reply:
yes